File: Web store/WebStoreApi/WebStore.Models/Entities/Storage.cs
using System.Collections.Generic;
using WebStore.Models.Models;

namespace WebStore.Models.Entities
{
    public class Storage : EntityBase<int>
    {
        public Storage()
        {
            Items = new HashSet<StorageItem>();
            Drones = new HashSet<Drone>();
        }

        public string Name { get; set; }

        public AddressCoordinates Coordinates { get; set; }

        public ICollection<StorageItem> Items { get; set; }

        public ICollection<Drone> Drones { get; set; }
    }
}

File: Web store/WebStoreWeb/src/models/storage.ts
import { StorageItem } from "./storage-item";
import { Drone } from "./drone";
import { AddressCoordinates } from "./address-coordinates";

export class Storage {
    public id: number;
    public name: string;
    public coordinates: AddressCoordinates;
    
    public items: StorageItem[];
    public drones: Drone[];
}